assets/js/search.js

```diff
@@ -0,0 +1,32 @@
+//Search Function
+var search = search || (function(){
+  var args = {};
+  return {
+    init : function(Args) {
+      args = Args;
+    },
+    
+    searchRows : function() {
+      var $rows = $(args[0] + " tr:not(:first)");
+      $("tr:nth-child(odd)").each(function() {
+        $(this).addClass("zebra-stripe");
+      });
+
+      $("#searchInput").keyup(function() {
+        var val = $.trim($(this).val()).replace(/ +/g, " ").toLowerCase();
+        $rows.show().filter(function() {
+          var text = $(this).text().replace(/\s+/g, " ").toLowerCase();
+          return !~text.indexOf(val);
+        }).hide();
+        
+        $(args[0] + " tr:visible").each(function(e) {
+          if (e % 2 == 1) {
+            $(this).addClass("zebra-stripe");
+          } else {
```

Comment:
The if-else statement is inconsistent with other ones.

Reply:
Oops, I will change the format of the if-else statement in login.html to follow this syntax.